pages/resources/opportunities.jsx

```diff
@@ -81,8 +89,12 @@ export default function Opportunities({ opps, ...props }) {
 const Wrapper = styled.div``;
 
 export async function getStaticProps() {
-  let opps = await prisma.posting.findMany();
-  opps = opps.sort((opp1, opp2) => (opp1.postedDate > opp2.postedDate ? 1 : -1));
+  let opps;
+  try {
+    opps = await prisma.posting.findMany();
+  } catch (e) {
+    opps = opps.sort((opp1, opp2) => (opp1.postedDate > opp2.postedDate ? 1 : -1));
```

Comment:
This is incorrect implementation. If the database fails, we use the cache you should import opportunities from ```cache/cache.js```

Outside of the try catch block then we sort and then we also filter by published = true since this is what gets pulled on the initial refresh whereas backend involves filters